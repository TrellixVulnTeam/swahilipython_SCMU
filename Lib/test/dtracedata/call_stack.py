eleza function_1():
    function_3(1, 2)

# Check stacktrace
eleza function_2():
    function_1()

# CALL_FUNCTION_VAR
eleza function_3(dummy, dummy2):
    pass

# CALL_FUNCTION_KW
eleza function_4(**dummy):
    rudisha 1
    rudisha 2  # unreachable

# CALL_FUNCTION_VAR_KW
eleza function_5(dummy, dummy2, **dummy3):
    ikiwa Uongo:
        rudisha 7
    rudisha 8

eleza start():
    function_1()
    function_2()
    function_3(1, 2)
    function_4(test=42)
    function_5(*(1, 2), **{"test": 42})

start()
