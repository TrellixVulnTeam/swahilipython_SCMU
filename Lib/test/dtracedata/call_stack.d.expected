 function-enjaribu:call_stack.py:start:23
 function-enjaribu: call_stack.py:function_1:1
 function-enjaribu:  call_stack.py:function_3:9
function-return:  call_stack.py:function_3:10
function-return: call_stack.py:function_1:2
 function-enjaribu: call_stack.py:function_2:5
 function-enjaribu:  call_stack.py:function_1:1
 function-enjaribu:   call_stack.py:function_3:9
function-return:   call_stack.py:function_3:10
function-return:  call_stack.py:function_1:2
function-return: call_stack.py:function_2:6
 function-enjaribu: call_stack.py:function_3:9
function-return: call_stack.py:function_3:10
 function-enjaribu: call_stack.py:function_4:13
function-return: call_stack.py:function_4:14
 function-enjaribu: call_stack.py:function_5:18
function-return: call_stack.py:function_5:21
function-return:call_stack.py:start:28
